src/components/clusterConfiguration/ClusterConfigurationForm.tsx

```diff
@@ -103,7 +104,7 @@ const ClusterConfigurationForm: React.FC<ClusterConfigurationFormProps> = ({
         return formikActions.setFieldError('name', `Name "${values.name}" is already taken.`);
       }
     } catch (e) {
-      console.error('Failed to perform unique cluster name validation.', e);
+      captureException('Failed to perform unique cluster name validation.', e);
```

Comment:
what is the right parameters order? you are missing string and error in different api calls to capture exceptions.

Reply:
good catch. Order was reversed. Fixed